src/debonair/Order.java

```diff
@@ -0,0 +1,125 @@
+// user defined package
+package debonair;
+
+// importing built in packages
+import java.time.LocalDateTime;
+import java.util.*;
+
+// importing classes from other packages
+import resources.Constants;
+
+public class Order {
+	// class variables
+	private int orderId;
+	private LocalDateTime orderDate = LocalDateTime.now();
+	private Set<Item> orderCart;
+	private static Map<Integer, Bill> currentSale = new HashMap<>();
+
+	// Getter-Setters
+	public int getOrderId() {
+		return orderId;
+	}
+
+	public void setOrderId(int orderId) {
+		this.orderId = orderId;
+	}
+
+	public LocalDateTime getOrderDate() {
+		return orderDate;
+	}
+
+	public void setOrderDate(LocalDateTime orderDate) {
+		this.orderDate = orderDate;
+	}
+
+	public Set<Item> getOrderCart() {
+		return orderCart;
+	}
+
+	public void setOrderCart(TreeSet<Item> orderCart) {
+		this.orderCart = orderCart;
+	}
+
+	public static Map<Integer, Bill> getCurrentSale() {
+		return currentSale;
+	}
+
+	public static void setCurrentSale(HashMap<Integer, Bill> currentSale) {
+		Order.currentSale = currentSale;
+	}
+
+	// Overridden toString method
+	@Override
+	public String toString() {
+		return "Order [orderId=" + orderId + ", orderDate=" + orderDate + ", orderCart=" + orderCart + "]";
+	}
+
+	// Parameterized constructor
+	public Order(int orderId, LocalDateTime orderDate, Set<Item> orderCart) {
+		super();
+		this.orderId = orderId;
+		this.orderDate = orderDate;
+		this.orderCart = orderCart;
+	}
+
+	/**
+	 * Function to create order of the customer
+	 */
+	public static void createOrder() {
+		Random randomObj = new Random();
```

Comment:
Creating a new Random object each time a random value is needed is inefficient and may produce numbers which are not random depending on the JDK. For better efficiency and randomness, create a single Random, then store, and reuse it.